docs/current/quickstart/snippets/build-multi/index.mjs

```diff
@@ -0,0 +1,40 @@
+import { connect } from "@dagger.io/dagger"
+
+connect(async (client) => {
+
+  // use a node:16-slim container
+  // mount the source code directory on the host
+  // at /src in the container
+  const source = await client.container()
+    .from("node:16-slim")
+    .withMountedDirectory('/src', client.host().directory('.', { exclude: ["node_modules/", "ci/"] }))
```

Comment:
move `client.host` to a variable so code is more readable?

Reply:
No need to await here. `directory()` is not a leaf